src/commands/bindle.rs

```diff
@@ -89,6 +87,22 @@ pub struct Push {
     )]
     pub bindle_server_url: String,
 
+    /// Basic http auth username for the bindle server
+    #[structopt(
+        name = BINDLE_HTTP_USER,
+        long = "bindle-http-user",
+        env = BINDLE_HTTP_USER,
```

Comment:
Not blocking this PR: we should make username and password environment variables consistent across commands. Like spin deploy requires a hippo user name and password and the env vars look a little diff there.

Reply:
Good point; updated in https://github.com/fermyon/spin/pull/407/commits/1fe296466b575a0dfe888775766b0a94e8b5cd62